docs/streams.md

```diff
@@ -98,105 +90,94 @@ example, if you had the following events in a stream :
 3@test-stream
 ```
 
-If you set the `TruncateBefore` or `$tb` value to 3, a read of the stream would result in only reading the
-last event:
+Setting the **`TruncateBefore`** (or **`$tb`**) value to 3, results in reading only the last event: 
 
 ```
 3@test-stream
 ```
 
-A **soft delete** makes use of `TruncateBefore` and `$tb`. When you delete a stream, its `TruncateBefore`
-or `$tb` is set to
-the [max long/Int64 value](https://docs.microsoft.com/en-us/dotnet/api/system.int64.maxvalue?view=net-5.0):
-9223372036854775807. When you read a soft deleted stream, the read returns a `StreamNotFound` or `404` result.
-After deleting the stream, you are able to append to it again, continuing from where it left off.
+A **soft delete** leverages **`TruncateBefore`** (or **`$tb`**). When you delete a stream, **`TruncateBefore`**
+(or **`$tb`**) is set to the [max long/Int64 value](https://docs.microsoft.com/en-us/dotnet/api/system.int64.maxvalue?view=net-5.0): 9223372036854775807. 
+
+Appending to the stream adds events starting from event number 4, the last stream revision before deletion, incremented by one.  
```

Comment:
This order is a little confusing. I would suggest:

> A **soft delete** leverages **`TruncateBefore`** (or **`$tb`**). When you delete a stream, **`TruncateBefore`**
(or **`$tb`**) is set to the [max long/Int64 value](https://docs.microsoft.com/en-us/dotnet/api/system.int64.maxvalue?view=net-5.0): 9223372036854775807. 

> If you attempt to read this soft-deleted stream the read returns a **`StreamNotFound`** or **`404`** result.

> Appending to the stream later adds events starting from the last stream revision before deletion, incremented by one. Continuing with our previous example, a new event would be appended with event number 4, and only events from event number 4 onwards are visible when you read this stream.